src/index.ts

```diff
@@ -32,6 +32,7 @@ export { SearchForm } from 'SearchForm/SearchForm';
 export { SearchPane } from 'SearchPane/SearchPane';
 export { Section } from 'Section/Section';
 export { SectionTitle } from 'SectionTitle/SectionTitle';
+export { default as SiteAlert } from 'SiteAlert';
```

Comment:
This is a slightly different method of exporting, but I think it's preferable and the other exports in this file should follow the same pattern:

- makes easier to change components' main export without touching this file (e.g. if SiteAlert/index.ts wants to `export default SiteAlertView` rather than `export default SiteAlert`, then we would not need to touch this file 🚀 
- prevents a linting error on the SiteAlert/index.ts because the export is never used (not imported elsewhere in the app)